<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1168</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Character Creation</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QWidget" name="leftSide" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="bodypart_buttons">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>520</width>
           <height>460</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>520</width>
           <height>10000</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="chest">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>110</y>
            <width>91</width>
            <height>151</height>
           </rect>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::DefaultContextMenu</enum>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="thigh_left">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>270</y>
            <width>31</width>
            <height>81</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="shin_left">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>360</y>
            <width>31</width>
            <height>81</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="thigh_right">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>270</y>
            <width>31</width>
            <height>81</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="hand_left">
          <property name="geometry">
           <rect>
            <x>440</x>
            <y>110</y>
            <width>31</width>
            <height>31</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="arm_left">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>110</y>
            <width>61</width>
            <height>31</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="forearm_left">
          <property name="geometry">
           <rect>
            <x>370</x>
            <y>110</y>
            <width>61</width>
            <height>31</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="head">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>30</y>
            <width>51</width>
            <height>61</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="shin_right">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>360</y>
            <width>31</width>
            <height>81</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="forearm_right">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>110</y>
            <width>51</width>
            <height>31</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="arm_right">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>110</y>
            <width>61</width>
            <height>31</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="hand_right">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>110</y>
            <width>31</width>
            <height>31</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>440</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="2" column="0">
           <widget class="QLabel" name="label_name">
            <property name="text">
             <string>Character Name:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="field_name"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_author">
            <property name="text">
             <string>Author Name:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="field_author"/>
          </item>
          <item row="6" column="1">
           <widget class="QPushButton" name="submit_button">
            <property name="text">
             <string>Finish and save</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="rightSide" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="PreviewWidget" name="previewWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="rightSideButtons" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer1">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>70</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="resetButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="randomizeButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Randomize</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionLoad_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="workingDirectoryAction">
   <property name="text">
    <string>Working Directory</string>
   </property>
   <property name="toolTip">
    <string>Change the Working Directory</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PreviewWidget</class>
   <extends>QWidget</extends>
   <header>previewwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>submit_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>finish_and_save_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>-5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chest</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>body_part_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>-4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>head</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>body_part_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>-8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>arm_left</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>body_part_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>-8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>forearm_left</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>body_part_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>-8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hand_left</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>body_part_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>-5</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>arm_right</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>body_part_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>-7</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>forearm_right</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>body_part_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hand_right</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>body_part_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>thigh_right</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>body_part_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>thigh_left</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>body_part_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shin_right</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>body_part_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shin_left</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>body_part_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>-2</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>body_part_clicked()</slot>
  <slot>tool_button_clicked()</slot>
  <slot>color_picker_clicked()</slot>
  <slot>zoom_in_clicked()</slot>
  <slot>zoom_out_clicked()</slot>
  <slot>mirror_button_clicked(bool)</slot>
  <slot>finish_and_save_clicked()</slot>
  <slot>pen_width_dragged(int)</slot>
 </slots>
</ui>
